cura/PrinterOutputDevice.py

```diff
@@ -74,6 +111,30 @@ def setTargetBedTemperature(self, temperature):
         self._target_bed_temperature = temperature
         self.targetBedTemperatureChanged.emit()
 
+    ## Time the print has been printing.
+    #  Note that timeTotal - timeElapsed should give time remaining.
+    @pyqtProperty(float, notify = timeElapsedChanged)
+    def timeElapsed(self):
+        return self._time_elapsed
+
+    ## Total time of the print
+    #  Note that timeTotal - timeElapsed should give time remaining.
+    @pyqtProperty(float, notify=timeTotalChanged)
+    def timeTotal(self):
+        return self._time_total
+
+    @pyqtSlot(float)
+    def setTimeTotal(self, new_total):
```

Comment:
eh. The setTimeTotal doesn't really need to be exposed. The property for getting time elapsed and time total are needed (as they are displayed)